weld/ast.rs

```diff
@@ -916,6 +916,18 @@ impl<T: TypeBounds> Expr<T> {
         }
     }
 
+    pub fn contains_symbol(&self, sym: &Symbol) -> bool {
```

Comment:
Add a comment saying what this is looking for. Maybe it's also better to call it contains_ident since it only looks at Ident?